.circleci/config.yml

```diff
@@ -425,6 +448,7 @@ workflows:
       - test
       - detekt
       - assemble-sample-app
+      - run-backend-integration-tests
```

Comment:
I think it should be fine to run these on all commits... They shouldn't take too long.